### PR TITLE
Call BSS web hook when an artist is updated.

### PR DESCRIPTION
**Pull request by bunnymatic**
problem
---------

we need to notify BryantStreetStudios when an artist changes their
profile data or their art.

solution
---------

add a webhook and ping it when artists change any data or art pieces
change.

### CODE REVIEW

File: app/services/bryant_street_studios_webhook.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class BryantStreetStudiosWebhook
+  def self.artist_updated(artist_id)
```
Comment (bunnymatic) [inline app/services/bryant_street_studios_webhook.rb:2]: Maybe down the road we filter this by artists who are at 1890.  b/c right now any artist update will ping. 1890 will happily delete nothing from the cache but it's more traffic than we actually need.

File: config/config.yml [hunk 2]
```diff
@@ -55,6 +55,7 @@ production:
   subdomain: openstudios
 
 common:
+  bryant_street_studios_webhook_url: http://localhost:9292/webhook
```
Comment (bunnymatic) [inline config/config.yml:58]: added the api key to `config.keys.yml` which is not checked in and managed in deployments.

**Comment by bunnymatic** [comment]
Still TODO for this PR.
Don't post if the web hook url or api key are not set - makes it so we can safely deploy without setting those things immediately.

File: app/services/bryant_street_studios_webhook.rb (hunk 1 above)
Comment (kneyzberg) [reply app/services/bryant_street_studios_webhook.rb:2]: Oh yeah that totally makes sense, so we should have a studio check

**Review by kneyzberg — APPROVED** [review]

**Pull request merged**